Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugMultiplayerMetrics.cpp

```diff
@@ -0,0 +1,222 @@
+            {
+                const MultiplayerStats::ComponentStats& componentStats = stats.m_componentStats[aznumeric_cast<AZStd::size_t>(netComponentId)];
+                for (AZStd::size_t index = 0; index < componentStats.m_propertyUpdatesSent.size(); ++index)
+                {
+                    const PropertyIndex propertyIndex = aznumeric_cast<PropertyIndex>(index);
+                    const char* propertyName = componentRegistry->GetComponentPropertyName(netComponentId, propertyIndex);
+                    const MultiplayerStats::Metric& subMetric = componentStats.m_propertyUpdatesSent[index];
+                    callsPerSecond = 0.0f;
+                    bytesPerSecond = 0.0f;
+                    AccumulatePerSecondValues(stats, subMetric, callsPerSecond, bytesPerSecond);
+                    DrawMetricsRow(propertyName, false, subMetric.m_totalCalls, subMetric.m_totalBytes, callsPerSecond, bytesPerSecond);
+                }
+                ImGui::TreePop();
+            }
+        }
+        {
+            const MultiplayerStats::Metric metric = stats.CalculateComponentPropertyUpdateRecvMetrics(netComponentId);
+            float callsPerSecond = 0.0f;
+            float bytesPerSecond = 0.0f;
+            AccumulatePerSecondValues(stats, metric, callsPerSecond, bytesPerSecond);
+            if (DrawMetricsRow("PropertyUpdates Recv", true, metric.m_totalCalls, metric.m_totalBytes, callsPerSecond, bytesPerSecond))
+            {
+                const MultiplayerStats::ComponentStats& componentStats = stats.m_componentStats[aznumeric_cast<AZStd::size_t>(netComponentId)];
+                for (AZStd::size_t index = 0; index < componentStats.m_propertyUpdatesRecv.size(); ++index)
+                {
+                    const PropertyIndex propertyIndex = aznumeric_cast<PropertyIndex>(index);
+                    const char* propertyName = componentRegistry->GetComponentPropertyName(netComponentId, propertyIndex);
+                    const MultiplayerStats::Metric& subMetric = componentStats.m_propertyUpdatesRecv[index];
+                    callsPerSecond = 0.0f;
+                    bytesPerSecond = 0.0f;
+                    AccumulatePerSecondValues(stats, subMetric, callsPerSecond, bytesPerSecond);
+                    DrawMetricsRow(propertyName, false, subMetric.m_totalCalls, subMetric.m_totalBytes, callsPerSecond, bytesPerSecond);
+                }
+                ImGui::TreePop();
+            }
+        }
+        {
+            const MultiplayerStats::Metric metric = stats.CalculateComponentRpcsSentMetrics(netComponentId);
+            float callsPerSecond = 0.0f;
+            float bytesPerSecond = 0.0f;
+            AccumulatePerSecondValues(stats, metric, callsPerSecond, bytesPerSecond);
+            if (DrawMetricsRow("RemoteProcedures Sent", true, metric.m_totalCalls, metric.m_totalBytes, callsPerSecond, bytesPerSecond))
```

Comment:
nit: Theres a little bit of repeated code here in each block, consider moving to helper function to avoid DRY. Might not be worth it due to differences in property blocks versus RPC but code looks like it could be refactored,